04.ls/lib/entry.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Entry
+  attr_reader :file_name
+
+  def initialize(path)
+    @file_name = File.basename(path)
+  end
+
+  def hidden?
+    @file_name =~ /^\..*/
```

Comment:
読みやすさの点では

```suggestion
    @file_name[0] != '.'
```

とか

```suggestion
    !@file_name.start_with?('.')
```

でもいいのでは？